language-support/java/BUILD.bazel

```diff
@@ -1,13 +1,13 @@
 # Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-load("@com_github_google_bazel_common//tools/javadoc:javadoc.bzl", "javadoc_library")
 load("@os_info//:os_info.bzl", "is_windows")
+load("//bazel_tools:javadoc_library.bzl", "javadoc_library")
 
-# FIXME: `javadoc_library` rule fails to find sources on Windows
```

Comment:
Is that fixed?

Reply:
actually, I don't know. I'll add the comment back. sorry

Reply:
Interestingly enough, the windows build fails because it cannot find sources for daml-lf archive 1.6, but I didn't change anything there :(
I guess I do the `if not is_windows else None` dance there as well?

Reply:
Presumably that’s what the comment was trying to say :) Yeah we don’t build any docs on windows atm so happy to disable this on windows.